### ISSUE TITLE
Display metadata on collection show page

### ISSUE THREAD

**Issue by jcoyne** [curationexperts/alexandria-v2#59]
Depends on #49

**Comment by val99erie** [comment]
I don't really have enough information about which metadata fields need to be displayed, so I'm going to call this story done.

For now, I hooked up the view to display the fields that the MODS importer currently imports.  After the next demo, we can get a better idea of all the fields that need to be displayed, and we can open a new story to add them in.
